storybooknative/src/SimpleCard/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow strict
+// TODO create PlatformStyleSheet
```

Comment:
This comment can probably be removed now.
```suggestion
```

Reply:
done